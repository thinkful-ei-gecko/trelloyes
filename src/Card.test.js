import React from 'react';
import ReactDOM from 'react-dom';
import * as renderer from 'react-test-renderer';
import Card from './Card.js';


describe('Card test', () => {
  // shadow test
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card key={1} title='sample title' content='sample content'/>, div);
    ReactDOM.unmountComponentAtNode(div);

  });

  // snapshot test
  it('renders same UI', () => {
    const tree = renderer.create(<Card key={1} title='sample title' content='sample content' />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});