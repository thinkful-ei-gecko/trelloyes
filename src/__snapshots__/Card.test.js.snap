// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card test renders same UI 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    sample title
  </h3>
  <p>
    sample content
  </p>
</div>
`;
